src/stream/src/executor/mview/materialize.rs

```diff
@@ -406,7 +411,7 @@ impl<S: StateStore> std::fmt::Debug for MaterializeExecutor<S> {
 
 /// A cache for materialize executors.
 pub struct MaterializeCache {
-    data: ExecutorCache<Vec<u8>, Option<CompactedRow>>,
```

Comment:
Any reason for not using bytes here? 🥺

Reply:
Just didn't want to involve too much diff on the streaming code in this PR. If it's better to use `Bytes` here, I can do the refactor by the way. @BugenZhao

Reply:
Just replaced some usages of `Vec<u8>` by `Bytes` in materialize buffer and cache. The only remaining usage is the key of `ExecutorCache`. The reason is that `lru::KeyRef<Bytes>` does not implement `Borrow<&[u8]>` and cannot query with `&[u8]`.